target/linux/ar71xx/files/arch/mips/ath79/mach-tl-wr840n-v2.c

```diff
@@ -0,0 +1,127 @@
+/*
+ *  TP-LINK TL-WR841N/ND v9/v11 / TL-WR842N/ND v3
```

Comment:
It's already done;)
No problem, just for clarification:
Every mach-*.c file results to one one mach-*.o file.
Only one mach-*.o file is linked to the kernel according to device type.
So, I woud say that many small mach-*.c files are better from the kernel size point of view.